prog/test/vm.rb

```diff
@@ -51,6 +53,61 @@ class Prog::Test::Vm < Prog::Test::Base
 
   label def ping_google
     sshable.cmd("ping -c 2 google.com")
+    hop_verify_io_rates
+  end
+
+  def get_iops
+    fio_ios_cmd = <<~CMD
+      sudo fio --filename=./f --size=100M --direct=1 --rw=randrw --bs=4k --ioengine=libaio \\
+              --iodepth=256 --runtime=4 --numjobs=1 --time_based --group_reporting \\
+              --name=test-job --eta-newline=1 --output-format=json
+    CMD
+
+    fio_ios_output = JSON.parse(sshable.cmd(fio_ios_cmd))
+    fio_ios_output["jobs"].first["read"]["iops"] + fio_ios_output["jobs"].first["write"]["iops"]
+  end
+
+  def get_read_bw_mbytes
```

Comment:
I recommend having `get_read_bw_bytes` and `get_write_bw_bytes` (instead of mbytes).  Then having `verify_io_rates` do the calculation and comparison.  This issue with this approach is you are truncating down here, and comparing against a multiplication in `verify_io_rates`.

I'm guessing we don't care about exact numbers, since we are multiplying by `1.2`, but I think it is more flexible to expose the exact rates.  The truncation doesn't matter much for high rates (who cares between 473 and 474), but matters a lot for low rates (think about a value of 0, which could mean no IO, or anything less than 1MB).

We may currently only be using this to enforce maximum bandwidth limits, but if we later decide to use this for monitoring/logging, having it return the actual bytes is best.